docs/data/date-pickers/custom-field/custom-field.md

```diff
@@ -125,30 +125,30 @@ The new accessible DOM structure will become compatible with Joy UI in the futur
 
 ## With a custom editing experience
 
-### Using an `Autocomplete`
+### Using an Autocomplete
 
 If your user can only select a value in a small list of available dates,
-you can replace the field with an `Autocomplete` listing those dates:
+you can replace the field with the `<Autocomplete />` from `@mui/material` to those dates:
 
 {{"demo": "PickerWithAutocompleteField.js", "defaultCodeOpen": false}}
 
-### Using a read-only `TextField`
+### Using a read-only Text Field
```

Comment:
Not sure if this was intentional, but did you mean to use 'Text Field' or 'TextField' (like in the text below)?

```suggestion
### Using a read-only TextField
```

Reply:
or according to this doc: https://www.notion.so/mui-org/Style-conventions-for-MUI-components-and-products-dff7d2d7d1ba4a4abd47f48cf345b800?pvs=4#1279b9e04d514d32a109b846a0945813

shouldn't we use 'text field' instead?

Reply:
We are describing the `<TextField />` from `@mui/material` here, not the generic concept of a text field, so I don't think "text field" would be appropriated.

I'm using "Text Field" because it's a reference to one our our product (from the core team), similar to the Data Grid or the Date Time Picker.

I'll ping Sam for a validation of this PR before merging :+1:

Reply:
Well, technically it is not a product ... I guess... no? 🤔

Reply:
We're describing our specific implementation (rather than the general concept), so I would style it as a proper noun: Text Field.